.github/workflows/benchmark.yaml

```diff
@@ -0,0 +1,42 @@
+name: Continuous Benchmark
+on:
+  push:
+    branches:
+      - main
+    tags:
+  pull_request:
```

Comment:
I don't know if this can work, it will mean that any contributor will be able to get write access to the repo. Maybe we should only run this on main?

Reply:
Yeah I need to separate that. If it is a pr then having a benchmark result on pr comment is still nice, but shouldn't update the page

Reply:
Maybe we can remove this for now and see what the result looks like on main.